blog/2022-08-30-update2-0-alpha.mdx

```diff
@@ -4,9 +4,6 @@ authors: [ttin, team-lumi]
 tags: [update, 2-0, alpha, luminescent]
 ---
 
-import { Spoiler } from 'react-spoiler-tag'
-import 'react-spoiler-tag/dist/index.css'
-
```

Comment:
There is no need to import in markdown because of `MDXComponents.js`

Docusaurus doc https://docusaurus.io/docs/markdown-features/react#mdx-component-scope
![image](https://user-images.githubusercontent.com/25965213/233207640-3eecb7f3-a378-426e-bc86-ecd7406ab54d.png)